recipes/kcov/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Use the same "template" than used for other recipes. For instance: https://github.com/conan-io/conan-center-index/tree/master/recipes/cereal/all/test_package